examples/default/src/App.tsx

```diff
@@ -20,8 +23,24 @@ import { QueryClient, QueryClientProvider } from 'react-query';
 const queryClient = new QueryClient();
 
 export const App: React.FC = () => {
+  const shouldSyncSession = (data: SessionMetadata) => {
+    if (data.launchType === LaunchType.cold) {
+      return true;
+    }
+    if (data.sessionDurationInSeconds > 20) {
+      return true;
+    }
+    if (data.OS === 'OS Level 34') {
+      return true;
+    }
+    return false;
+  };
+
   const navigationRef = useNavigationContainerRef();
+
   useEffect(() => {
+    SessionReplay.setSyncCallback((data) => shouldSyncSession(data));
```

Comment:
Is it normal to call setSyncCallback before using the init method?

Reply:
actually that was recommended by native team so it catches any early sessions.